app/styles/_mobile-clients.less

```diff
@@ -0,0 +1,9 @@
+.mobile-clients-view {
+
+  .copy-to-clipboard {
```

Comment:
We should do this as one declaration if there are no other rules for `mobile-clients-view`

```less
.mobile-clients-view .copy-to-clipboard {
  /* ... */
}
```